### ISSUE TITLE
Add table of content

### ISSUE THREAD

**Issue by peterdesmet** [ranges-network/ranges-network.github.io#10]
Two (long) pages could benefit from a TOC on the side to make it more navigatable:

https://github.com/ranges-network/ranges-network.github.io/blob/main/pages/awards.md
https://github.com/ranges-network/ranges-network.github.io/blob/main/pages/privacy.md

Just add `toc: true` to the front matter:

```
layout: home
title: Privacy Policy
description: Our policy for the use of cookies and similar technologies and how , and for what purposes, we may collect and process your personal information when you visit the Ranges web site.
permalink: /privacy/
toc: true
```

**Comment by dbloom** [comment]
A good idea @peterdesmet. Implementation has caused an error (below). 

I added to privacy.md but didn't see a change after 14 mins, still no change. Then attempted to addd to add to awards.md, also no change.  After another 10 mins I received the following message. I have since removed the toc from both pages and tried again, only with awards.md. Same result, but with no error message (10 mins and counting...).

...

 Logging at level: debug Configuration file: /github/workspace/./_config.yml Theme: minima Theme source: /usr/local/bundle/gems/minima-2.5.1 GitHub Pages: github-pages v228 GitHub Pages: jekyll v3.9.3 Theme: minima Theme source: /usr/local/bundle/gems/minima-2.5.1 Requiring: jekyll-feed Requiring: jekyll-seo-tag Requiring: jekyll-feed Requiring: jekyll-sitemap Requiring: jekyll-coffeescript Requiring: jekyll-commonmark-ghpages Requiring: jekyll-gist Requiring: jekyll-github-metadata Requiring: jekyll-paginate Requiring: jekyll-relative-links Requiring: jekyll-optional-front-matter Requiring: jekyll-readme-index Requiring: jekyll-default-layout Requiring: jekyll-titles-from-headings Requiring: jekyll-remote-theme GitHub Metadata: Initializing... Source: /github/workspace/. Destination: /github/workspace/./_site Incremental build: disabled. Enable with --incremental Generating... Theme: peterdesmet/petridish@3.0 Theme source: /tmp/jekyll-remote-theme-20231101-8-4t60k1 Remote Theme: Using theme peterdesmet/petridish Remote Theme: Downloading https://codeload.github.com/peterdesmet/petridish/zip/3.0 to /tmp/jekyll-remote-theme-20231101-8-1ckrguj.zip Remote Theme: Unzipping /tmp/jekyll-remote-theme-20231101-8-1ckrguj.zip to /tmp/jekyll-remote-theme-20231101-8-4t60k1 EntryFilter: excluded /Gemfile EntryFilter: excluded /README.md Generating: JekyllOptionalFrontMatter::Generator finished in 0.000185811 seconds. Generating: JekyllReadmeIndex::Generator finished in 0.00354811 seconds. Jekyll Feed: Generating feed for posts Generating: JekyllFeed::Generator finished in 0.001125767 seconds. Generating: Jekyll::JekyllSitemap finished in 0.000771646 seconds. Generating: Jekyll::Paginate::Pagination finished in 6.801e-06 seconds. Generating: JekyllRelativeLinks::Generator finished in 0.001822607 seconds. Generating: JekyllDefaultLayout::Generator finished in 0.000253815 seconds. Generating: JekyllTitlesFromHeadings::Generator finished in 4.9203e-05 seconds. Rendering: pages/about.md Pre-Render Hooks: pages/about.md Rendering Markup: pages/about.md Requiring: kramdown-parser-gfm Rendering Layout: pages/about.md Layout source: theme GitHub Metadata: Generating for ranges-network/ranges-network.github.io GitHub Metadata: Calling @client.pages("ranges-network/ranges-network.github.io", {}) Rendering: pages/archive.md Pre-Render Hooks: pages/archive.md Rendering Markup: pages/archive.md Rendering Layout: pages/archive.md Layout source: theme Rendering: pages/awards.md Pre-Render Hooks: pages/awards.md Rendering Markup: pages/awards.md Rendering Layout: pages/awards.md Layout source: theme Rendering: pages/home.md Pre-Render Hooks: pages/home.md Rendering Markup: pages/home.md Rendering Layout: pages/home.md Layout source: theme Rendering: pages/participants.md Pre-Render Hooks: pages/participants.md Rendering Markup: pages/participants.md Rendering Layout: pages/participants.md Layout source: theme Rendering: pages/privacy.md Pre-Render Hooks: pages/privacy.md Rendering Markup: pages/privacy.md Rendering Layout: pages/privacy.md Layout source: theme Rendering: pages/team.md Pre-Render Hooks: pages/team.md Rendering Markup: pages/team.md Rendering Layout: pages/team.md Layout source: theme Rendering: assets/theme/css/main.scss Pre-Render Hooks: assets/theme/css/main.scss Rendering Liquid: assets/theme/css/main.scss Rendering Markup: assets/theme/css/main.scss github-pages 228 | Error: String can't be coerced into Integer Remote Theme: Cleaning up /tmp/jekyll-remote-theme-20231101-8-4t60k1 
...

**Comment by dbloom** [comment]
UPDATE:

I had to move on with other dev issues. As a result my toc request was cancelled, but a good 15 minutes after I made the initial commit.

Here is the error summary: https://github.com/ranges-network/ranges-network.github.io/actions/runs/6723140384

**Comment by peterdesmet** [comment]
There are two issues:

1. The pages `awards.md` and `participants.md` still have the `layout: home`. That layout (typically reserved for the homepage) does not accept a table of content. Removing that line allows to enable `toc: true`. I also noticed that the `background` properties were indented by 1 space while 2 are recommended (but that should not have affected it). I have implemented those changes in #12
2. The error in the run, which I will discuss in #11

**Comment by dbloom** [comment]
That's interesting. Those page use `layout:home` because when I named them something else, the banner disappeared on the page.

I will accept your pull request updating `home.md` but is it better practice to name/use `"participants" for participants.md`, and so on? Should I reassign those layouts even though you've updated `home`?

**Comment by dbloom** [comment]
@peterdesmet I understand why the banner vanished, I think,  re: my comment above.

So is it better to set the `layout:` for each page to something other than "home"?

**Comment by peterdesmet** [comment]
No, only set a `layout` for pages that deviate from the default (i.e. `home`, `team`, etc.). By default, pages without a `layout` will use the standard/default layout (i.e. a `layout: default`).

Definitely don’t set an empty `layout:`, that will cause the banner etc. to disappear.

**Comment by peterdesmet** [comment]
See the documentation at http://peterdesmet.com/petridish/docs/configuration/#pages for details.

**Comment by dbloom** [comment]
Will review again. I think the use of the `layout:` is based on the GBIF portal set up. I will try to keep those habits over there with that template.

Thank you for clarifying.
